test/PeNet.Test/Header/Resource/ResourcesTest.cs

```diff
@@ -185,7 +185,7 @@ public void IconsAsRaw_GivenPeFiles_IconsParsed(string file, int expectedIcons)
         public void IconsAsRaw_GivenPeFiles_IconsInGroupsParsed(string file, params int[] expectedIconsInGroups)
```

Comment:
Same test as test in line 224.
(IconsAsICO_GivenPeFiles_IconsInGroupsParsed)
[can be removed like IconsAsRaw_GivenPeFilesAndIconNotIncludedInPeFile_FoundIconDoNotContainsGivenIcon]